src/components/SearchError.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useEffect } from 'react';
+import { Typography } from '@mui/material';
+import { ShowData } from '../types';
+import { getMovieTrending } from '../helpers';
+import ShowCarousel from './ShowCarousel';
+
+interface ShowErrorProps {
+    /**
+     * String of query searched by user
+     */
+    query: string;
+}
+
+/**
+ * Rendered when MDB returns no results for a search.
+ *
+ * @returns {JSX.Element} | Search Error
+ */
+const SearchError: React.FC<ShowErrorProps> = ({ query }): JSX.Element => {
```

Comment:
For some reason this one did not save on my last review. I think that this screen is not actually an error screen, it is an expected state when no data is returned from the query. As such I think a name like EmptySearchResults or SearchResultsEmptyState makes more sense. 

Regardless of what name we choose, I think the prop name should match. If the component is `SearchError` the props should be `SearchErrorProps`.

Reply:
@Thenlie In your example `EmptySearchResults`, are you referencing the name of the function `SearchError`?

Reply:
Yes that is correct, the component name.
```suggestion
const EmptySearchResults: React.FC<EmptySearchResultsProps> = ({ query }): JSX.Element => {
```